EyeDrive/App.xaml.cs

```diff
@@ -1,8 +1,6 @@
 ﻿using Microsoft.Shell;
-using Squirrel;
```

Comment:
Please don't change code outside of the scope of the changes you are making to change branding and add a UWP project.

Reply:
https://github.com/Squirrel/Squirrel.Windows/issues/1102